Queries.md

```diff
@@ -0,0 +1,419 @@
+
+```sql
+with part AS (
+    SELECT  args->1#>'{hex}' AS participant1, args->2#>'{hex}' AS participant2,  args->0#>'{num}' AS channel_id
+    FROM "event"
+    WHERE event = 'ChannelOpened' AND address = '0xa5C9ECf54790334B73E5DfA1ff5668eB425dC474' 
+)
+SELECT participants, channel_id
+    FROM 
+    (
+        SELECT participant1 AS participants, channel_id
+        FROM part
+        union
+        SELECT participant2 AS participants, channel_id
+        FROM part
+        order BY participants
+    ) part_table
+    GROUP BY participants
+```
+
+
+## Participant AND the specific channel he opened
+
+```sql
+with part AS (
+    SELECT  args->1#>'{hex}' AS participant1, args->2#>'{hex}' AS participant2,  args->0#>'{num}' AS channel_id
+    FROM "event"
+    WHERE event = 'ChannelOpened' AND address = '0xa5C9ECf54790334B73E5DfA1ff5668eB425dC474' 
+)
+SELECT participants, channel_id
+FROM 
+(
+	SELECT participant1 AS participants, channel_id
+	FROM part
+	union
+	SELECT participant2 AS participants, channel_id
+	FROM part
+	order BY participants
+) part_table
+```
+
+## Network capacity
+
+
+``Network capacity = sum of all deposited amounts for the channels that are still opened``
+
+```sql 
+Select sum(channel_capacity)
+FROM 
+(
+    SELECT (args->0#>'{num}')::numeric AS channel_id, (args->2#>'{num}')::numeric AS channel_capacity
+    FROM "event"
+    WHERE event = 'ChannelNewDeposit' AND address = '0xa5C9ECf54790334B73E5DfA1ff5668eB425dC474'
+) channels_and_their_deposits
+WHERE channel_id NOT IN 
+(
+    SELECT channel_id 
+    FROM
+    (
+        SELECT (args->0#>'{num}')::numeric AS channel_id 
+        FROM "event"
+        WHERE event = 'ChannelOpened' AND address = '0xa5C9ECf54790334B73E5DfA1ff5668eB425dC474'
+    ) channel_opened_data
+    WHERE channel_id NOT IN 
+    (
+        SELECT (args->0#>'{num}')::numeric AS channel_id
+        FROM "event"
+        WHERE event = 'ChannelClosed' AND address = '0xa5C9ECf54790334B73E5DfA1ff5668eB425dC474' 
+    )
+)
+```
+
+
+## The first three channels with the highest capacity
+
+```sql
+Select channel_id, channel_capacity
+FROM 
+(
+    SELECT (args->0#>'{num}')::numeric AS channel_id, (args->2#>'{num}')::numeric AS channel_capacity
+    FROM "event"
+    WHERE event = 'ChannelNewDeposit' AND address = '0xa5C9ECf54790334B73E5DfA1ff5668eB425dC474'
+) channels_and_their_deposits
+WHERE channel_id NOT IN 
+(
+    SELECT channel_id 
+    FROM
+    (
+        SELECT (args->0#>'{num}')::numeric AS channel_id 
+        FROM "event"
+        WHERE event = 'ChannelOpened' AND address = '0xa5C9ECf54790334B73E5DfA1ff5668eB425dC474'
+    ) channel_opened_data
+    WHERE channel_id NOT IN 
+    (
+        SELECT (args->0#>'{num}')::numeric AS channel_id
+        FROM "event"
+        WHERE event = 'ChannelClosed' AND address = '0xa5C9ECf54790334B73E5DfA1ff5668eB425dC474' 
+    )
+)
+order BY channel_capacity DESC
+LIMIT 3
+```
+
+
+## The first three channels with the lowest capacity
```

Comment:
**The first three channels with the lowest capacity** a date histogram query would be nice just like the previous one